auth-web/src/stores/staff.ts

```diff
@@ -248,6 +250,19 @@ export const useStaffStore = defineStore('staff', () => {
     return {}
   }
 
+  async function searchReviews (filterParams: ReviewFilterParams) {
+    const response = await BusinessService.searchReviews(filterParams)
```

Comment:
If the Axios function experiences an error (eg, 403, 404, 5xx, etc) then this call will throw an exception.

I recommend adding a chained catch statement that returns, say, null, eg
`.catch(() => null) // return null on error`

Reply:
Updated

Reply:
Your update is OK but next time you can keep the code simpler by catching the exception as per my example.